.github/workflows/bump-homebrew-formula.yml

```diff
@@ -1,28 +1,55 @@
 name: Bump Homebrew formula for CLI
-
 on:
-  # It cannot run on release event as when release is created then version is not yet bumped in package.json
-  # This means we cannot extract easily latest version and have a risk that package is not yet on npm
-  push:
-    branches:
-      - master
+  schedule:
+    - cron: "0 23 * * *"
 
 jobs:
   bump-formula-in-homebrew:
     name: Bump the formula in homebrew-core repo
     if: startsWith(github.event.commits[0].message, 'chore(release):')
     runs-on: ubuntu-latest
     steps:
-      - name: Checkout repo
+      - name: Checkout repository
         uses: actions/checkout@v2
+
       - name: Get version from package.json
         id: extractver
-        run: echo "::set-output name=version::$(npm run get-version --silent)"
+        run: echo "::set-output name=version::$(node -p 'require("./package.json").version')"
+
+      - name: Check for changes in Homebrew formula
```

Comment:
what are we checking here exactly? it checks if brew formula was updated, but what does it tell us exactly?